tests/integration/conftest.py

```diff
@@ -148,19 +146,31 @@ def modal_dialog(mocker, homedir):
 
 
 @pytest.fixture(scope="function")
-def print_dialog(mocker, homedir):
+def export_service():
+    """An export service that assumes the Qubes RPC calls are successful and skips them."""
+    export_service = export.Service()
+    # Ensure the export_service doesn't rely on Qubes OS:
+    export_service._run_disk_test = lambda dir: None
+    export_service._run_usb_test = lambda dir: None
+    export_service._run_disk_export = lambda dir, paths, pasphrase: None
+    export_service._run_printer_preflight = lambda dir: None
+    export_service._run_print = lambda dir, paths: None
+    return export_service
+
```

Comment:
There is a little bit of duplication here (this fixture is present in `tests/conftest.py`). It is not immediately clear to me why the unit and integration tests define distinct fixtures, and I was a little bit surprized to find out that importing fixtures is not trivial. I time-boxed it and ended up concluding it wasn't worth it extracting this one to a separate file.

_As a side note, I'm not sure how I feel about the PyTest fixtures facilities (magic?). I started with Ruby and used to love that kind of syntactic sugary stuff, then worked with Go for some time and I don't anymore. Truth is I find that not being able to import fixtures like you'd import any other Python function is a bit of a smell, but I must recognize that I haven't researched to much into it._